etc/config/analysis.amazon.properties

```diff
@@ -10,3 +10,14 @@ compiler.llvm-mcatrunk.name=llvm-mca (trunk)
 compiler.llvm-mcatrunk.exe=/opt/compiler-explorer/clang-trunk/bin/llvm-mca
 # Sets both input and output to Intel variant
 compiler.llvm-mcatrunk.intelAsm=--x86-asm-syntax=intel -output-asm-variant=1
+
+group.osaca.compilers=osacatrunk
+group.osaca.groupName=Open Source Architecture Code Analyzer
+group.osaca.supportsBinary=false
+group.osaca.demangler=/opt/compiler-explorer/gcc-9.1.0/bin/c++filt
+group.osaca.compilerType=osaca
+
+compiler.osacatrunk.name=OSACA (trunk)
+compiler.osacatrunk.exe=/opt/compiler-explorer/osaca-trunk/bin/osaca
```

Comment:
You could use `pip install git+https://github.com/RRZE-HPC/OSACA.git` to install directly from our master branch. The only down side of this, is that it will take about two minutes, as it builds caches for all architectures to speed up the first run. If you're uncomfortable with this, use 0.3.9 for now.

Reply:
Got it: 0.3.9 works best: we don't have too good a situation for handling "trunk" tools if they update right now. We can change it later